README.md

```diff
@@ -102,6 +112,47 @@ void main(List<String> args) async {
 }
 ```
 
-
 ## Contributing
+
 Please contribute! If you want to fix a bug, suggest improvements, or add new features to the project, just [open an issue](https://github.com/hbtalha/genius_lyrics/issues/new) or send me a pull request.
+
+### Environment Variables
```

Comment:
Please move this section to right below the `A complete example` section

Reply:
I dont think it should be there, since its part of the setup for contributing with this project.

Reply:
`Contributing` usually is one the last sections in a README, that's why I am requesting to move the `Environment Variables` section up before the merge.

The README structure would be like this:
## Setup

## Usage

## A complete example

## Environment Variables

## Contributing

Reply:
Sorry, I thought you're saying to put inside the example section. Sure, I'll adjust this.